dot/peerset/peerset.go

```diff
@@ -318,7 +318,7 @@ func (ps *PeerSet) updateTime() error {
 				// forget peer removes the peer from the list of members of the set.
 				err = ps.peerState.forgetPeer(set, peerID)
 				if err != nil {
-					return fmt.Errorf("cannot forget peer %s from set %d: %w", peerID, set, err)
+					return fmt.Errorf("cannot forget peer: %w", err)
```

Comment:
nit add the set context in the `forgetPeer` error `fmt.Errorf("%w: for peer id %s", ErrPeerDoesNotExist, peerID)`